scripts/bash/pr_options.sh

```diff
@@ -8,17 +8,21 @@ pr_template() {
   TARGET_BRANCH="main"
 
   USER_NAME=$(git config user.name)
-  REPO_NAME=$(basename -s .git `git config --get remote.origin.url`) # your repo name, can be fork name
+  URL=$(git config --get remote.origin.url)
+  REPO_NAME=$(basename -s .git "$URL") # your repo name, can be fork name
+  ORGANISATION="$(echo "$URL" | sed -r 's/(.+):(.+)\/([^.]+)(\.git)?/\2/')"
```

Comment:
We usually prefer American spelling, so that would make this "ORGANIZATION" instead